docs/source/models/concept_models/vivarium_nutrition_optimization/concept_model.rst

```diff
@@ -131,6 +131,80 @@ Step 1 in this process is detailed in the :ref:`pregnancy<2021_concept_model_viv
 
   More specific emulator documentation
 
+
+2.2 Submodels
+-------------
+
+.. list-table:: Wave I outstanding tasks for costing and emulator
+  :header-rows: 1
+
+  * - Task
+    - Dependencies
+    - RT person
+    - Note
+  * - 1: Confirm mixed coverage approach 
+    - 
+    - Ali 
+    - Completed 
+  * - 2: Emulator design choice 
+    - 
+    - Full team 
+    - 
+  * - 3: Clean emulator and remove hardcoded information 
+    - 
+    - Syl 
+    - 
+  * - 4: Allow for optimization to multiple outcomes 
+    - 
+    - TBD
+    - Example: deaths, DALYs, SAM cases 
+  * - 5: Add maternal interventions 
+    - 
+    - TBD
+    - 
+  * - 6: Test for robustness to different initial values 
+    - 
+    - TBD
+    - 
+  * - 7: Add non-linear cost functions 
```

Comment:
Should we have this in the wave I or wave II section?

Reply:
wave I, if possible

Reply:
Ok, I've been thinking about this as not-necessary-for-success in the wave I interim deliverable given that Laura and Rahul said they were interested in just product costs only to start, but agreed that it would be cool to add it in if we can do so without blocking wave I finalization otherwise. Is that what you're thinking?

Reply:
I agree it is not needed for a successful wave 1. I was thinking of this as testing the emulator with dummy non-linear cost data to make sure it works as expected. Also forces us to build the emulator in such a way that this would be possible. But it could move to wave 2 if we think it will block us.

Reply:
I added a note stating it wasn't technically needed and changed the language a bit. Hopefully this works for everyone now!

Reply:
Awesome, sounds good to me!